spec/features/create_etd_spec.rb

```diff
@@ -139,8 +132,7 @@
     # provide copyright permissions letters/files
     expect(page).not_to have_content(permissions_filename)
     expect(page).to have_selector('#pbPermissionFilesUploaded', visible: :hidden)
-    permissions_upload_input = file_upload_elements[11]
```

Comment:
This was the line that was breaking due to upping the number of supplemental inputs on the form. Turns out referring to an input using an array index rather than an ID is pretty brittle!